src/DataGrid.tsx

```diff
@@ -1008,6 +1021,7 @@ function DataGrid<R, SR, K extends Key>(
           rowIdx,
           row,
           viewportColumns: rowColumns,
+          isRowSelectable: isRowSelectable?.(row),
```

Comment:
Could enforce a boolean/default.
```suggestion
          isRowSelectable: isRowSelectable?.(row) ?? true,
```